main.go

```diff
@@ -155,8 +156,8 @@ func main() {
 			Target: "http://" + listener.Addr().String(),
 		})
 		if err != nil {
-			fmt.Println("err", err)
-			panic(err)
+			logger.Error("Failed to create route", zap.Error(err), zap.String("path", apiPath))
+			continue
```

Comment:
Why continue loading the server? This hides server loading issues, and it takes a while for whomever ran this service to know something is off (as the process didn't exit)